salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -0,0 +1,104 @@
+include:
+  - ...deployed.namespace
+
+Create fluent-bit ConfigMap:
+  metalk8s_kubernetes.object_present:
+    - manifest:
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: fluent-bit
+          namespace: metalk8s-logging
+          labels:
+            app: fluent-bit
+            app.kubernetes.io/managed-by: salt
+            app.kubernetes.io/name: fluent-bit
+            app.kubernetes.io/part-of: metalk8s
+            heritage: metalk8s
+            release: fluent-bit
+        data:
+          fluent-bit.conf: |-
+            [SERVICE]
+                HTTP_Server    On
+                HTTP_Listen    0.0.0.0
+                HTTP_PORT      2020
+                Flush          1
+                Daemon         Off
+                Log_Level      warn
+                Parsers_File   parsers.conf
+            [INPUT]
+                Name           tail
+                Tag            kube.*
+                Path           /var/log/containers/*.log
+                Parser         docker
+                DB             /run/fluent-bit/flb_kube.db
+                Mem_Buf_Limit  5MB
+            [INPUT]
+                Name              systemd
+                Tag               host.*
+                DB                /run/fluent-bit/flb_journal.db
```

Comment:
Minor: I know we do not really care, but look like indentation is not the same as the other block

Reply:
Right, but even the configuration generated from helm chart is not consistent with indentation. :D
I'll change it.